system-test/testnet-automation.sh

```diff
@@ -98,6 +99,8 @@ function launch_testnet() {
         -p "$TESTNET_TAG" $maybePublicIpAddresses --dedicated \
         ${ADDITIONAL_FLAGS[@]/#/" "}
       ;;
+    bare)
```

Comment:
Not clear how to use this option. Shall I create config/config with a set of bare metal IPs?

Reply:
I will setup the pipeline later hehe.

Reply:
waiting for follow up prs!